Modules/zlibmodule.c

```diff
@@ -1416,7 +1416,9 @@ zlib_adler32_impl(PyObject *module, Py_buffer *data, unsigned int value)
     } else {
         value = adler32(value, data->buf, (unsigned int)data->len);
     }
-    return PyLong_FromUnsignedLong(value & 0xffffffffU);
+
+    assert(sizeof(value) == 4 && value >= 0);
```

Comment:
Can we really assert that `sizeof(unsigned int)` is always 4? Wikipedia (https://en.wikipedia.org/wiki/C_data_types#Main_types) claims it can be 2 bytes.

Reply:
I use the current style, `& 0xffffffffU`, it's more clear.